index.js

```diff
@@ -1437,6 +1437,21 @@
     }
   };
 
+  //# not :: Boolean -> Boolean
+  //.
+  //. Takes a boolean value and negates that value.
```

Comment:
"Boolean" should be capitalized, as it's named after a person.

I'd like to be explicit about what the function returns:

_Takes a Boolean and returns the negation of that value (`false` for `true`; `true` for `false`)._